packages/react-storage/src/components/StorageBrowser/Views/Controls/Paginate.tsx

```diff
@@ -163,20 +163,23 @@ const getButtonVariantProps = (
   { variant, ...props }: ButtonElementProps,
   context: PaginateStateContext
 ): ButtonElementProps => {
-  const [{ hasNext, hasPrevious, isLoadingNextPage }, handleUpdateState] =
-    context;
+  const [
+    { hasNext, hasPrevious, isLoadingNextPage, current },
+    handleUpdateState,
+  ] = context;
 
-  let ariaDisabled, ariaLabel, className, disabled, onClick;
+  let ariaCurrent: AriaAttributes['aria-current'];
```

Comment:
Typescript had a hard time implicitly discerning what type this should be so I set it explicitly here. Probably because of the specific strings it allows? 
`boolean | "false" | "true" | "page" | "step" | "location" | "date" | "time"`